### DISCUSSION TITLE
[Bug]: Almost impossible to zoom out on Mac

### DISCUSSION THREAD

**Discussion by terryma** [blakeblackshear/frigate#12353] [Report a Bug] [answered]
### Describe the problem you are having

I'm on a Macbook and I can double-click on the live stream to zoom in, but zooming out is almost impossible with the trackpad. When a mouse is connected zooming out with the scroll wheel works. 

### Steps to reproduce

- Go to any live view or recorded events
- Double click to zoom in
- Try zooming out with the trackpad and struggle

### Version

0.14.0-4e5a6eb

### Frigate config file

```yaml
N/A
```

### Relevant log output

```shell
N/A
```

### Operating system

Debian

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

amcrest

**Comment by hawkeye217** [comment, marked as answer]
I don't have an issue on my Macbook Pro in Safari or Firefox, though pinch-to-zoom definitely is slower than just dragging two fingers up or down the trackpad.

This can be adjusted with some code, but just to be sure that's all that's needed, what browser are you using and what gesture are you using on the trackpad to zoom out?

**Reply by NickM-27** [reply]
Same on my Mac, using the typical scroll gesture works as expected

**Reply by terryma** [reply]
Haha by complete accident I managed to figure this out on my own today. Is this a standard gesture? I've always been using pinch to zoom this entire time.

**Reply by hawkeye217** [reply]
Yep, I do believe it's a standard gesture.

Nonetheless, I did put up a PR (https://github.com/blakeblackshear/frigate/pull/12364) that was merged and will be available in beta 4 that increases the sensitivity of the pinch-to-zoom feature. So you should be able to use that gesture as well.
